docs/modules/SDDs/pages/getting-started.adoc

```diff
@@ -0,0 +1,265 @@
+= Getting Started with Project Syn
+
+This guide helps to get started with the various Project Syn tools.
+
+== Requirements
+
+Before you start, please make sure to have these requirements available:
+
+* A local Kubernetes cluster (k3s), managed with https://k3d.io/[k3d]
+* A https://gitlab.com[GitLab.com] account with an https://gitlab.com/profile/keys[SSH key configured]
+* `curl`
+
+== Kickstart Lieutenant
+
+Create a namespace to host the Operator and the API
+[source,shell]
+----
+kubectl create namespace lieutenant
+----
+
+=== Install Lieutenant Operator
+
+Install *Lieutenant Operator* with the following commands:
+
+[source,shell]
+----
+# CRDs (global scope)
+kubectl apply -k github.com/projectsyn/lieutenant-operator/deploy/crds
+
+# Operator deployment
+kubectl -n lieutenant apply -k github.com/projectsyn/lieutenant-operator/deploy
+
+# Configure Lieutenant Operator to not rely on Hashicorp Vault (for now)
+kubectl -n lieutenant set env deployment/lieutenant-operator -c lieutenant-operator SKIP_VAULT_SETUP=true
+----
+
+=== Install Lieutenant API
+
+Install *Lieutenant API* with the following commands:
+
+[source,shell]
+----
+# API deployment
+kubectl -n lieutenant apply -k github.com/projectsyn/lieutenant-api/deploy
+
+# Ingress
+export K3S_INGRESS_IP=$(kubectl -n kube-system get svc traefik -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+kubectl -n lieutenant apply -f -<<EOF
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: lieutenant-api
+spec:
+  rules:
+  - host: lieutenant.${K3S_INGRESS_IP}.nip.io
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: lieutenant-api
+          servicePort: 80
+EOF
+----
+
+Check that the API is accessible:
+
+[source,shell]
+----
+curl http://lieutenant.${K3S_INGRESS_IP}.nip.io/healthz
+----
+This should return `ok` as the answer to the `curl` command.
+
+TIP: The API documentation can be accessed in your browser under http://lieutenant.${K3S_INGRESS_IP}.nip.io/docs.
+
+=== Prepare Lieutenant Operator access to GitLab.com
```

Comment:
"For Lieutenant to create new Git repos for newly created/added clusters it needs API access to a GitLab server. This is currently not optional." ?

Reply:
added